src/main/java/org/sagebionetworks/web/client/jsinterop/mui/Grid.java

```diff
@@ -0,0 +1,91 @@
+package org.sagebionetworks.web.client.jsinterop.mui;
+
+import org.sagebionetworks.web.client.jsinterop.ReactComponentType;
+import org.sagebionetworks.web.client.jsinterop.react.HasStyle;
+
+public class Grid extends HasStyle<ReactComponentType<GridProps>, GridProps> {
```

Comment:
Re-add Grid widget (but it's unused for now, so no risk if this still doesn't work)